fabtests/pytest/efa/test_rdm.py

```diff
@@ -24,18 +24,36 @@ def run_rdm_test(cmdline_args, executable, iteration_type, completion_type, memo
                           pytest.param("standard", marks=pytest.mark.standard)])
 def test_rdm_pingpong(cmdline_args, iteration_type, completion_type, memory_type):
     run_rdm_test(cmdline_args, "fi_rdm_pingpong", iteration_type,
-                 completion_type, memory_type)
+            completion_type, memory_type, "all")
+
+@pytest.mark.parametrize("message_size",
+                         [pytest.param("r:8000,4,9000", marks=pytest.mark.short)])
```

Comment:
I do not think we need the "short" marker here. With the "short" marker, the test will no be run when user specify "-t standard".

Reply:
Isn't this the idea to run these message sizes with short marker only? We don't want to enable them with long iterations.